perception/ground_segmentation/src/scan_ground_filter_nodelet.cpp

```diff
@@ -168,7 +170,13 @@ void ScanGroundFilterComponent::classifyPointCloud(
 
       float global_slope = std::atan2(p->orig_point->z, p->radius);
       // check points which is far enough from previous point
-      if (global_slope > global_slope_max_angle) {
+      if (
+        (height_from_gnd < -split_height_distance_) ||
+        ((p->orig_point->z - ground_cluster.getAverageHeight()) < -split_height_distance_) ||
+        (height_from_gnd > non_ground_heigh_thresh_)) {
+        // exclude pcl under ground or higher than limit
+        calculate_slope = false;
```

Comment:
In this case, what would be the label for point?

Reply:
I assigned as out_range label for that points

Reply:
When the point is labeled out of range, prev point label [will be also set](https://github.com/badai-nguyen/autoware.universe/blob/fix/ground_filter/perception/ground_segmentation/src/scan_ground_filter_nodelet.cpp#L231) to out of range. Is this okay for next point judgement?

Reply:
as [this condition ](https://github.com/badai-nguyen/autoware.universe/blob/c786f6b5c62fe488430222b64394cd12fe6492b5/perception/ground_segmentation/src/scan_ground_filter_nodelet.cpp#L191) , if the point is labeled out of range, the next point should not classify as POINT_FOLLOW, but GROUND, NON_GROUND or OUT_OF_RANGE. 
Keeping to use the current ground_cluster and non_ground_cluster to check the next point is reasonable, IMO.